app/views/errors/index.html.haml

```diff
@@ -72,3 +73,5 @@
             %label
               %input{:type => "checkbox", :value => ""}/
               Windows 7
+
+= render 'websites/website_modal'
```

Comment:
de ce trebuie sa fie in errors doar modalu asta de website? in errors.coffee ai codul de adaugare de website? Cred ca ti-ar fi mai simplu sa faci un controller sau sa folosesti controlleru de websites (websites_controller) si sa faci new website sa fie modalu tau. Pana la urma nu vrem sa avem 2 formuri pentru adaugare de website. O sa iti fac un task acum si vreau sa schimbi functionalitatea asta intr-un alt PR..!!!